OpenEdXMobile/res/drawable/ic_open_in_new.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="11dp"
+    android:height="11dp"
+    android:viewportWidth="12"
+    android:viewportHeight="12">
+    <path
+        android:fillColor="@color/neutralXDark"
```

Comment:
Please change the icon color to `neutalXXDark` to match the 'Open in Browser' color. 
I think we can also revert the icon size back to 12dp because it's not aligning anyway. :joy_cat:

Reply:
> Please change the icon color to neutalXXDark to match the 'Open in Browser' color.
the icon color is `707070` in Figma designs.

> I think we can also revert the icon size back to 12dp because it's not aligning anyway. 😹
DOne.